src/lib/features/client-feature-toggles/client-feature-toggle.controller.ts

```diff
@@ -180,10 +182,39 @@ export default class FeatureController extends Controller {
                     featuresSize + segmentsSize,
                 );
 
-                await this.clientFeatureToggleService.getClientDelta(
-                    undefined,
-                    query!,
+                const delta =
```

Comment:
do you want to run this sequentially or in parallel with the query you compare it against?